ui/src/shared/components/Sidebar/Sidebar.scss

```diff
@@ -30,7 +30,7 @@
 	.o-sidebar__nav-item-sublink > a {
 		padding: 6px 0 6px 44px;
 		display: block;
-		background-color: rgba(255, 255, 255, 0.04);
+		background-color: rgb(255 255 255 / 4%);
```

Comment:
Maybe declare a variable for this color.
It is also used in ui/src/react-admin/modules/shared/components/Sidebar/Sidebar.scss